src/server/scripts/Northrend/Naxxramas/instance_naxxramas.cpp

```diff
@@ -420,6 +417,9 @@ class instance_naxxramas : public InstanceMapScript
 
             void Update(uint32 diff) override
             {
+                if (!instance->HavePlayers())
+                    return;
+
```

Comment:
Is this needed with the trigger now being responsible for spawning the waves?

Reply:
now it seems unnecessary